docs/source/intervention_models/maternal_supplementation/index.rst

```diff
@@ -107,48 +111,66 @@ Given the low utilization of MMS and BEP relative to IFA, we assume that baselin
     - Note
   * - Ethiopia
     - Pregnant population
-    - Percent who took antenatal iron for 90+ days
-    - 10.6
+    - Proportion who took *any* antenatal iron
+    - 0.598 (0.583, 0.613), normal distribution of uncertainty clipped between zero and one
+    - Use this value for the :ref:`acute malnutrition simulation <2019_concept_model_vivarium_ciff_sam>`; DHS 2019
+  * - Ethiopia
+    - Pregnant population
+    - Proportion who took antenatal iron for 90+ days
+    - 0.106
     - DHS 2019
   * - India
     - Pregnant population
-    - Percent who took antenatal iron for 90+ days
-    - 38.7
+    - Proportion who took antenatal iron for 90+ days
+    - 0.387
     - DHS 2017
   * - Mali
     - Pregnant population
-    - Percent who took antenatal iron for 90+ days
-    - 28.0
+    - Proportion who took antenatal iron for 90+ days
+    - 0.28
     - DHS 2017
   * - Pakistan
     - Pregnant population
-    - Percent who took antenatal iron for 90+ days
-    - 29.4
+    - Proportion who took antenatal iron for 90+ days
+    - 0.294
     - DHS 2017
   * - Tanzania
     - Pregnant population
-    - Percent who took antenatal iron for 90+ days
-    - 21.4
+    - Proportion who took antenatal iron for 90+ days
+    - 0.214
     - DHS 2017
 
 .. note::
 
   DHS has coverage data specific to women who took iron tablets for <60 and 60-89 days too as well as stratified by age, residence, region, education, and wealth quintile.
 
+  The baseline percent of women taking any antenatal iron during pregnancy (59.8%) was chosen instead of the percent of women who took antenatal iron during pregnancy for 90 or more days (10.6%) to be conservative and due to the following finding in [Pena-Rosas-et-al-2015]_: 
+
+    "Overall, for women receiving *any* intermittent iron regimen (with or without other vitamins and minerals) compared with a daily regimen there was no clear evidence of differences between groups for any infant primary outcomes: low birthweight (average risk ratio (RR) 0.82; 95% confidence interval (CI) 0.55 to 1.22; participants = 1898; studies = eight; low quality evidence), infant birthweight (mean difference (MD) 5.13 g; 95% CI ‐29.46 to 39.72; participants = 1939; studies = nine; low quality evidence), premature birth (average RR 1.03; 95% CI 0.76 to 1.39; participants = 1177; studies = five; low quality evidence), or neonatal death (average RR 0.49; 95% CI 0.04 to 5.42; participants = 795; studies = one; very low quality). None of the studies reported congenital anomalies." (Abstract)
+
+  Note that the :ref:`BEP simulation <2017_concept_model_vivarium_gates_bep>` used the proportion of women who took antenatal iron for 90+ days for baseline coverage.
+
+  The confidence interval was calculated using the formula :math:`1.96 \times \sqrt{\frac{p \times (1 - p)}{n}}`.
```

Comment:
What is the distribution of your data here?

Reply:
I noted in the table above that we should assume a normal distribution of uncertainty within the confidence interval around the sample estimate of the proportion. The equation/theory behind this is described on this page. https://bolt.mph.ufl.edu/6050-6052/module-9/sampling-distribution-of-p-hat/

Does that answer your question?

Reply:
Yes!

Reply:
Great, thanks!!